QModManager/API/SMLHelper/MonoBehaviours/Fixer.cs

```diff
@@ -0,0 +1,56 @@
+namespace QModManager.API.SMLHelper.MonoBehaviours
+{
+    using UnityEngine;
+    using System.Reflection;
+    using Logger = API.SMLHelper.Logger;
+
+    public class Fixer : MonoBehaviour, IProtoEventListener
+    {
+        [SerializeField]
+        public TechType techType;
+
+        [SerializeField]
+        public string ClassId;
+
+        private float time;
+        private bool initalized;
+
+        private static FieldInfo BuilderPrefab = typeof(Builder).GetField("prefab", BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
Publicized assembly...?

Reply:
Yes. SMLHelper used the publicized assembly wherever possible.

Reply:
This could be easily converted to use publicized assemblies